subprojects/docs/src/docs/userguide/migrating_from_ant.adoc

```diff
@@ -0,0 +1,328 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[migrating_from_ant]]
+= Migrating Builds From Apache Ant
+
+https://ant.apache.org/[Apache Ant] is a build tool with a long history in the Java world and it is still widely used. While flexible, it lacks conventions and many of the powerful features that Gradle can provide. So migrating to Gradle is often worth the effort.
+
+The biggest problem with migrating from Ant to Gradle is that there is no such thing as a standard Ant build! That makes it difficult to provide specific instructions. Fortunately, Gradle has some great integration features with Ant that can make the process smoother. And even migrating from https://ant.apache.org/ivy/[Ivy]-based dependency management isn't particularly hard because Gradle has a similar model based on <<dependency_management_terminology#sub:terminology_configuration,dependency configurations>> and it works with Ivy-compatible repositories.
+
+We will start by outlining the kinds of approaches you can take to migrating from Ant and offer some general guidelines.
+
+== General guidelines
+
+When you undertake to migrate a build from Ant to Gradle, you should keep in mind the nature of both what you already have and where you would like to end up. Do you want a Gradle build that mirrors the structure of the existing Ant build? Or do you want to move to something that is more idiomatic to Gradle? This is an important decision that will impact the difficulty of the migration and the benefits you accrue from switching to Gradle.
+
+To understand the implications, it may be useful to look at the two extremes:
+
+ 1. Using <<ant.html#sec:import_ant_build,`ant.importBuild()`>> to import the Ant build wholesale into Gradle.
++
+This approach is quick and simple and works for many Ant-based builds. Your targets become Gradle tasks that you can run, and the dependencies between targets are retained.
++
+The downside is that you lose the advantages of Gradle's conventions, many of its plugins, and many of its important features like incremental build, proper multi-project support, the build cache, and more.
+ 2. Creating the Gradle build from scratch.
++
+This sounds intimidating, but you may be surprised at how much heavy lifting Gradle's plugins can do. You will often find that large swathes of an Ant build are unnecessary in the equivalent Gradle one. The big advantage of this approach is that you gain access to all of Gradle's most important features.
++
+The downside is that many builds will have behavior that can't be replicated by existing plugins, so the migration will take time and effort. You also need a good understanding of what the Ant build is doing in order to match the behavior in Gradle accurately.
```

Comment:
My understanding is: that it depends and we can reuse a bunch. Gradle can import ant tasks, so maybe better to have an inflection of hope here?

There seems to be a bunch of good content to link to under `Migrating multi-project builds` below.